### PR TITLE
improving the readability of some code

### PR DESCRIPTION
**Pull request by onlymellb**
### What problem does this PR solve? 
Improve the readability of the code for processing `LastTransitionTime`

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes
 - Has Go code change

### Does this PR introduce a user-facing change?:
 
 ```release-note
None
 ```

### CODE REVIEW

File: pkg/manager/member/tidb_member_manager.go [hunk 1]
```diff
@@ -387,12 +387,13 @@ func (tmm *tidbMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, se
 			Health: health,
 		}
 		oldTidbMember, exist := tc.Status.TiDB.Members[name]
+
+		newTidbMember.LastTransitionTime = metav1.Now()
```
Comment (gregwebs) [inline pkg/manager/member/tidb_member_manager.go]: This looks like a bugfix instead of just a readability change?

Comment (onlymellb) [reply pkg/manager/member/tidb_member_manager.go]: Not a bugfix, the logic is still the same as before

**Comment by weekface** [comment]
@xiaojingchen ptal

**Pull request merged**